src/main/java/com/google/api/codegen/go/GoDiscoveryContext.java

```diff
@@ -41,6 +41,21 @@ public GoDiscoveryContext(Service service, ApiaryConfig apiaryConfig) {
           .put(Field.Kind.TYPE_DOUBLE, "0.0")
           .build();
 
+  @Override
+  protected String arrayTypeName(String elementName) {
```

Comment:
Go is in some sense a special case here, so I'll add comments to that effect. The logic is identical across all languages, so it seems redundant to create another set of identically-structured functions.